src/routes/auth/login.ts

```diff
@@ -1,23 +1,27 @@
 import jws from 'jws'
 export async function login (req, res): Promise<void> {
-  const username = req.body.username
-  const password = req.body.password
-  if (username && password) {
+  if (req.mpsService.config.jwt_secret && req.mpsService.config.web_admin_user && req.mpsService.config.web_admin_password) {
```

Comment:
this shouldn't be here -- these not being set should prevent server start up.

Reply:
Yes, we do that as well. Here is just an extra check.

Reply:
done